mypyc/test-data/run-integers.test

```diff
@@ -328,3 +328,14 @@ def test_int_as_bool() -> None:
     for x in 1, 55, -1, -7, 1 << 50, 1 << 101, -(1 << 50), -(1 << 101):
         assert is_true(x)
         assert not is_false(x)
+
+[case testMin]
+def check_min_int() -> None:
+    x: int = 200
+    y: int = 30
+    assert min(x, y) == 30
```

Comment:
Also test the case where the first argument is the minimum. I think that it's going to fail.

To help debug this, also add an irbuild test case for this. This way you can validate that the code you are generating is what you expect it to be.

Reply:
I get why you think that way. I've updated the logic a bit to get a better IR. I tested locally and the other case seem to work fine as well. I'll add that test case before pushing next time.